Makefile

```diff
@@ -23,6 +23,13 @@ COMMA:=,
 
 export CGO_ENABLED=0
 
+export CI ?= no
```

Comment:
Teamcity job configuration sets `CI=yes` and store the Google application credentials at `/teamcity/%teamcity.build.id%-credentials.json` file. So when this job is run from teamcity, it will mount `/teamcity` directory otherwise locally it will mount `$(HOME)/gcp` directory.